integration_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -Eemo pipefail
+# This script is meant to be run inside https://hub.docker.com/_/docker
+# See .github/workflows/ci.yml:18 for how to run this yourself
+
+# Wait for the docker daemon to start up
+for i in {1..30}; do
+    if docker ps; then
+        break
+    else
+        sleep 1
+    fi
+done
+if [ "$i" = '30' ]; then
+    echo "Docker did not start within 30 seconds, aborting"
+    exit 1
+fi
+
+# Start docuum in the background
+/artifacts/docuum-x86_64-unknown-linux-musl --threshold=13MB &
+sleep 1
+# Wait for docuum to start sleeping by checking the process state
+function wait {
```

Comment:
`wait` is a shell builtin, so I'd prefer if we called this something else (e.g., `wait_for_docuum`).

Reply:
Was wondering why `wait` was being highlighted in my editor haha, good call!